main.go

```diff
@@ -113,6 +115,15 @@ func parseConfig() (spirev1alpha1.ControllerManagerConfig, ctrl.Options, error)
 		if err := spirev1alpha1.LoadOptionsFromFile(configFileFlag, scheme, &options, &ctrlConfig); err != nil {
 			return ctrlConfig, options, fmt.Errorf("unable to load the config file: %w", err)
 		}
+
+		for _, ignoredNamespace := range ctrlConfig.IgnoreNamespaces {
+			regex, err := regexp.Compile(ignoredNamespace)
+			if err != nil {
+				return ctrlConfig, options, fmt.Errorf("unable to compile ignore namespaces regex: %w", err)
+			}
+
+			ignoreNamespacesRegex = append(ignoreNamespacesRegex, regex)
```

Comment:
nit: instead of creating a global var, what do you think about returning this regex list on parseConfig? and pass it to required functions?